### PR TITLE
Add papers from pldi'21, icse'21, oakland'21

### PR DESCRIPTION
**Pull request by luxroot**
Add papers from pldi'21, icse'21, oakland'21

### CODE REVIEW

File: data/fuzzers.json [hunk 1]
```diff
@@ -787,6 +866,31 @@
...
+    ],
+    "targets": [
+      "CPU"
+    ],
+    "toolurl": "https://github.com/compsec-snu/difuzz-rtl",
+    "color": "greybox",
+    "keywords": [
+      "Register-coverage"
+    ]
+  },
```
Comment (sangkilc) [inline data/fuzzers.json:893]: Add reference to "classming"

File: data/fuzzers.json [hunk 2]
```diff
@@ -2390,6 +2545,25 @@
...
+    "title": "NTFUZZ: Enabling Type-Aware Kernel Fuzzing on Windows with Static Binary Analysis",
+    "booktitle": "Proceedings of the IEEE Symposium on Security and Privacy",
+    "miscurl": [
+      "https://softsec.kaist.ac.kr/~jschoi/data/oakland2021.pdf"
+    ],
+    "targets": [
+      "Kernel"
+    ],
+    "color": "greybox"
+  },
```
Comment (sangkilc) [inline data/fuzzers.json:2566]: Add "references" to IMF

File: data/fuzzers.json [hunk 3]
```diff
@@ -2663,6 +2837,32 @@
...
+    "booktitle": "Proceedings of the IEEE Symposium on Security and Privacy",
+    "miscurl": [
+      "https://faculty.ist.psu.edu/wu/papers/polyglot.pdf"
+    ],
+    "targets": [
+      "File"
+    ],
+    "toolurl": "https://github.com/s3team/Polyglot",
+    "references": [
+      "AFL"
```
Comment (sangkilc) [inline data/fuzzers.json:2862]: Not closely related to AFL. CSmith and DIE are more relevant here.

File: data/fuzzers.json [hunk 4]
```diff
@@ -3236,6 +3461,29 @@
...
+    "toolurl": "https://kilthub.cmu.edu/articles/software/SyRust_Artifact_PLDI2021_Artifact/14356976",
+    "title": "SyRust: Automatic Testing of Rust Libraries with Semantic-Aware Program Synthesis",
+    "booktitle": "Proceedings of the ACM Conference on Programming Language Design and Implementation",
+    "miscurl": [
+      "https://doi.org/10.1184/R1/14356949"
+    ],
+    "keywords": [
+      "Rust"
+    ]
+  },
```
Comment (sangkilc) [inline data/fuzzers.json:3486]: Need to add a reference here too. Do you have a PDF for this?

Comment (luxroot) [reply data/fuzzers.json:3486]: There's a [direct link](https://kilthub.cmu.edu/ndownloader/files/27557852) at https://doi.org/10.1184/R1/14356949

And for reference, it's hard to find which fuzzer SyRust heavily relies on. Maybe RESTler?

Comment (sangkilc) [reply data/fuzzers.json:3493]: RESTler should be it.

File: data/fuzzers.json [hunk 5]
```diff
@@ -436,6 +459,34 @@
...
+    ],
+    "toolurl": "https://github.com/NWU-NISL-Fuzzing/COMFORT",
+    "title": "Automated Conformance Testing for JavaScript Engines via Deep Compiler Fuzzing",
+    "booktitle": "Proceedings of the ACM Conference on Programming Language Design and Implementation",
+    "miscurl": [
+      "https://arxiv.org/pdf/2104.07460.pdf"
+    ],
+    "keywords": [
+      "JavaScript"
+    ]
```
Comment (sangkilc) [inline data/fuzzers.json:488]: Add `references`. Maybe the three js fuzzers they compared in the paepr because DeepFuzz is not in the database.

Comment (luxroot) [reply data/fuzzers.json:488]: I thought the references should be DeepFuzz or none. There are four other fuzzers in comparison (not heavily relies on) and at CONTRIBUTING.md we limit the number of references to 2. So what should be reference for this?

Comment (sangkilc) [reply data/fuzzers.json:488]: Adding the three JS fuzzers should be okay in this case.

Comment (sangkilc) [reply data/fuzzers.json:494]: In fact, the most relevant one should be Montage in this case. So leaving only one reference is preferrable.

**Pull request merged**